### PR TITLE
Add 'docker_custom_udp_ports' and 'docker_udp_allow' variables

### PR DESCRIPTION
**Pull request by dddpaul**
Docker services expose not only TCP ...

### CODE REVIEW

**Comment by drybjed** [comment]
I knew that I should wait before tagging this... Anyway, are IP addresses expected to be the same for TCP and UDP? Because in that case the rules could be easily combined into one.

**Comment by drybjed** [comment]
Actually, is Docker even using UDP traffic at all? When I check my `docker` instance, it seems to only listen for TCP... What gives?

**Comment by dddpaul** [comment]
Well, Docker daemon uses only TCP by itself. But some dockerized services like Consul use UDP.
I expect that permitted IP address must be the same for UDP and TCP.

I've tried to combine this in one rule but failed. What `procotol` value must be in this case? `both` or `['tcp', 'udp']` didn't work.

**Comment by drybjed** [comment]
`protocol: [ 'tcp', 'udp' ]` should do the trick, make sure that you have recent `debops.ferm` role.

File: defaults/main.yml [hunk 1]
```diff
@@ -275,10 +275,10 @@ docker_ferm_dependent_rules:
...
-    dport: '{{ [ docker_tcp_port ] + docker_custom_tcp_ports }}'
+    dport: '{{ [ docker_tcp_port ] + docker_custom_ports }}'
+    protocol: [ 'tcp', 'udp' ]
     saddr: '{{ docker_tcp_allow }}'
     accept_any: False
     weight: '50'
     role: 'docker'
-    name: 'rest_api_rules'
-
+    name: 'docker_services_rules'
```
Comment (drybjed) [inline defaults/main.yml]: Change this to `name: 'service_rules'`, it will be combined with the `role` parameter to `50_docker_service_rules`.

**Comment by dddpaul** [comment]
`services_rules` is ok? Or `service_rules` is better?

**Comment by drybjed** [comment]
I think `service_rules` sounds better in English. The rules of the Docker service.

**Pull request merged**